src/vmm/src/pipe.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Sartura All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+#![cfg(target_arch = "x86_64")]
```

Comment:
I don't think this is needed, the top-level crate configuration is already set to `x86_64`-exclusive.